### DISCUSSION TITLE
Usuario no existe

### DISCUSSION THREAD

**Discussion by neido6** [aclap-dev/video-downloadhelper#362] [Help and Q&A]
### Your question

Como puedo tener soporte ya que la licencia que compre en el 2020 de manera vitalicia no aparece

### Addon & CoApp version and details

Ayer intente descargar un video y me pide una llave, al intentar crear la llave con mi correo me dice que no existe, la ultima vez que descargue un video con el plugin fue hace 6 meses.

**Comment by jcfrog** [comment]
hi,
please read this: https://github.com/aclap-dev/video-downloadhelper/wiki/Old-Chrome-licenses-(key-missing)
